tests/foreman/ui/test_host.py

```diff
@@ -338,25 +341,25 @@ def module_libvirt_subnet(
 
 
 @pytest.fixture(scope='module')
-def module_libvirt_lce(module_org):
+def module_lce(module_org):
     return entities.LifecycleEnvironment(organization=module_org).create()
 
 
 @pytest.fixture(scope='module')
-def module_libvirt_product(module_org):
+def module_product(module_org):
     return entities.Product(organization=module_org).create()
 
 
 @pytest.fixture(scope='module')
-def module_libvirt_repository(os_path, module_libvirt_product):
+def module_repository(os_path, module_product):
     repo = entities.Repository(
-        product=module_libvirt_product, url=os_path).create()
+        product=module_product, url=os_path).create()
     call_entity_method_with_timeout(entities.Repository(id=repo.id).sync, timeout=3600)
     return repo
 
 
 @pytest.fixture(scope='module')
-def module_libvirt_media(module_org, module_loc, os_path, module_libvirt_os):
+def module_libvirt_media(module_org, module_loc, os_path, module_os):
```

Comment:
will this name also be changed to module_media? instead module_libvirt_media

Reply:
As of now this being used only for libvirt and need not to generalized it. Whenever other tests will be added and if they want to generalize it they can.

For GCE that function is not required.

Reply:
okay make sense